packages/sled/index.js

```diff
@@ -61,6 +61,28 @@ function getNestedHandler(handlerString) {
   }
 }
 
+/**
+ * Given an inputs and the file name of a schema, validates the input.
+ * @param {String} input The input, config, or output to be validated
+ * @param {String} schemaFile The relative path to the schema
+ * @returns {Promise} A Promise resolving to true if Input is valid or rejecting if it errors
+ */
+function validateMessage(input, schemaFile) {
+  if (input && schemaFile) {
+    return readJsonFile(schemaFile)
+      .then((schema) => {
+        const ajv = new Ajv();
+        const schemaValidator = ajv.compile(schema);
```

Comment:
This can be removed if you use `const valid = ajv.validate(schema, input);`

Reply:
I liked having the object schemaValidator because I could pull the error details out of it 
schemaValidator.errors

Reply:
This is the example in the docs:

```
// ...
var valid = ajv.validate(schema, data);
if (!valid) console.log(ajv.errors);
// ...
```

It looks like you can pull the errors from `ajv` in that case.  I don't really have a strong opinion on this one.  If you're happy then I'm happy.